### PR TITLE
Added predict_aop_data function

### PR DESCRIPTION
**Pull request by satsin06**
Download the data using retrive_aop_data and predict the tiles using deepforest model

### CODE REVIEW

File: neonwranglerpy/lib/predict_aop_data.py
Comment (MarconiS) [inline neonwranglerpy/lib/predict_aop_data.py:1]: the funciton still has a couple of issues, but they are downstream issues related to retrieve_aop, likely, so I am ok with it to be merged with the base code.

(1) Ln 62 has a -1000 that is a bandaid on the fact that the tiles that gets previously downloaded is shifted 1000m east. This needs to be solved and the -1000 from this line needs to be removed

(2) there are a few prints that we needed for debugging but we don't need anymore (e.g. ln 42,64,67, 78).

Good job with your very first PR!

**Review by MarconiS — APPROVED** [review]

File: neonwranglerpy/lib/predict_aop_data.py
Comment (satsin06) [reply neonwranglerpy/lib/predict_aop_data.py:1]: ThankYou so much @MarconiS

**Comment by Nageshbansal** [comment]
Congrats @satsin06 for your first PR being merged!!

**Comment by satsin06** [comment]
> Congrats @satsin06 for your first PR being merged!!

Thank you @Nageshbansal 😊

**Pull request merged**